.devcontainers/OWNERS

```diff
@@ -0,0 +1,17 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
Also, this entire directory should be `.devcontainer` not `.devcontainers`

Reply:
I think either is fine. Is there a particular reason you would want this or is it a nit?

Just because in the future we could have more than one.

Reply:
I think `.devcontainer` is the appropriate name as that is consistent with the directory containing the `devcontainer.json` spec.

Reply:
vscode doesn't pick up the directory if it's plural